int64 IsReset
---
string FeedbackMsg
